### ISSUE TITLE
Bug: Inference fails with "llama_get_logits_ith: invalid logits id 7, reason: no logits" in ollama

### ISSUE THREAD

**Issue by yurivict** [ggerganov/llama.cpp#8911]
### What happened?

Inference fails with this cryptic error.

This happens with both CPU and Vulkan engines.

What might be causing this?

### Name and Version

llama-cpp-3538
ollama-0.3.4

### What operating system are you seeing the problem on?

BSD

### Relevant log output

```shell
[GIN] 2024/08/07 - 10:01:31 | 200 |  7.589808394s |       127.0.0.1 | POST     "/api/chat"
time=2024-08-07T10:01:31.521-07:00 level=DEBUG source=sched.go:462 msg="context for request finished"
time=2024-08-07T10:01:31.521-07:00 level=DEBUG source=sched.go:334 msg="runner with non-zero duration has gone idle, adding timer" modelPath=/home/yuri/.ollama/models/blobs/sha256-ff82381e2bea77d91c1b824c7afb83f6fb73e9f7de9dda631bcdbca564aa5435 duration=5m0s
time=2024-08-07T10:01:31.521-07:00 level=DEBUG source=sched.go:352 msg="after processing request finished event" modelPath=/home/yuri/.ollama/models/blobs/sha256-ff82381e2bea77d91c1b824c7afb83f6fb73e9f7de9dda631bcdbca564aa5435 refCount=0
time=2024-08-07T10:01:52.804-07:00 level=DEBUG source=sched.go:571 msg="evaluating already loaded" model=/home/yuri/.ollama/models/blobs/sha256-ff82381e2bea77d91c1b824c7afb83f6fb73e9f7de9dda631bcdbca564aa5435
DEBUG [process_single_task] slot data | n_idle_slots=4 n_processing_slots=0 task_id=1 tid="0x236209412000" timestamp=1723050112
time=2024-08-07T10:01:52.854-07:00 level=DEBUG source=routes.go:1346 msg="chat request" images=0 prompt="[INST] Say something.[/INST] "
DEBUG [process_single_task] slot data | n_idle_slots=4 n_processing_slots=0 task_id=2 tid="0x236209412000" timestamp=1723050112
DEBUG [launch_slot_with_data] slot is processing task | slot_id=0 task_id=3 tid="0x236209412000" timestamp=1723050112
DEBUG [update_slots] slot progression | ga_i=0 n_past=0 n_past_se=0 n_prompt_tokens_processed=8 slot_id=0 task_id=3 tid="0x236209412000" timestamp=1723050112
DEBUG [update_slots] kv cache rm [p0, end) | p0=0 slot_id=0 task_id=3 tid="0x236209412000" timestamp=1723050112
llama_get_logits_ith: invalid logits id 7, reason: no logits
time=2024-08-07T10:01:53.403-07:00 level=DEBUG source=server.go:1048 msg="stopping llama server"
time=2024-08-07T10:01:53.403-07:00 level=DEBUG source=server.go:1054 msg="waiting for llama server to exit"
time=2024-08-07T10:01:53.951-07:00 level=DEBUG source=server.go:1058 msg="llama server stopped"
```

**Comment by compilade** [comment]
Could you give clear instructions on how to reproduce this problem?

At a minimum, the command-line parameters given to `llama-server` would be useful.
